docker-compose.linux-dev.yaml

```diff
@@ -1,8 +1,35 @@
 version: "3.2"
```

Comment:
@alexvpickering Is there any reason why we cannot use a minimalistic `docker-compose.linux-dev.yaml` with only the overrides/additions to `docker-compose.yaml`, like the one already in https://github.com/biomage-ltd/worker/blob/develop/docker-compose.linux-dev.yaml

This works by running `docker-compose -f docker-compose.yaml -f docker-compose.linux-dev.yaml ...`. If using `make`, the `Makefile` should already been taking care of this.